.github/renovate.json

```diff
@@ -2,5 +2,6 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "local>axonivy/renovate-config"
-  ]
+  ],
+  "commitMessagePrefix": ""
```

Comment:
Ok, now we have proof that this doesn't work/provide the effect of a reproducible PR title. So I'll look further

https://github.com/axonivy/web-tester/pull/357
![again-chore-prs](https://github.com/user-attachments/assets/adb656c2-31bc-467a-ae9e-1718894fd625)